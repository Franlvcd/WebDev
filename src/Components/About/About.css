.about {
  display: flex;
  text-align: center;
  vertical-align: center;
  padding: 50px 0;
  justify-content: center;
}

.background {
  padding: 24px;
  min-height: 700px;
}

.title-text-about,
.vision-text {
  font-family: 'Roboto Slab', cursive;
  font-size: 50px;
  color: black;
  text-align: center;
  font-weight: bold;
  /* padding-bottom: 48%; */
  /* margin-top: 20px; */
  /* margin-bottom: 5px; */
}

.title-text-about {
  margin-top: 60px;
}

.vision-text {
  margin-top: 35px;
}

.background-vision {
  background-color: white;
  height: 320px;
  padding-top: 25px;
}

.about-content {
  color: black;
  text-align: center;
  width: 75%;
  margin: auto;
  margin-bottom: 50px;
  font-size: 30px;
  font-family: 'Roboto Slab', cursive;
}

@media screen and (max-width: 480px) {
  .about-content {
    color: black;
    text-align: center;
    width: 100%;
    margin: auto;
    margin-bottom: 50px;
    font-size: 30px;
    font-family: 'Roboto Slab', cursive;
  }
}

@media screen and (max-width: 700px) {
  .about-content {
    color: black;
    text-align: center;
    width: 100%;
    margin: auto;
    margin-bottom: 50px;
    font-size: 20px;
    font-family: 'Roboto Slab', cursive;
  }
}

@media screen and (max-width: 1000px) {
  .about-content {
    color: black;
    text-align: center;
    width: 75%;
    margin: auto;
    margin-bottom: 50px;
    font-size: 20px;
    font-family: 'Roboto Slab', cursive;
  }
}
