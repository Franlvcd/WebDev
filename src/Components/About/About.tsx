import './About.css'
import { Typography } from 'antd'
import { Divider } from 'antd'
import { Image } from 'antd'

const { Title } = Typography

export const About = () => {
  return (
    <div
    // className="about"
    // style={{
    // padding: 24,
    // minHeight: 700,
    // backgroundImage: `url("https://cdn.seeklearning.com.au/media/images/career-guide/module/environmental-engineer-module.jpg")`,
    // backgroundPosition: 'center',
    // backgroundSize: 'cover',
    // backgroundRepeat: 'no-repeat',
    // }}
    >
      <div
        style={{
          padding: 0,
          minHeight: 700,
          backgroundImage: `url("https://www.britannica.com/explore/savingearth/wp-content/uploads/sites/4/2019/03/environmental-engineering-hero.jpg")`,
          backgroundPosition: 'center',
          backgroundSize: 'cover',
          backgroundRepeat: 'no-repeat',
        }}
      >
        {' '}
      </div>
      <div>
        <p style={{ fontSize: '80px', color: 'black', marginTop: '20px' }}>
          Future is in your subscription
        </p>
      </div>

      <div
        style={{
          padding: 24,
          // minHeight: 700,
          backgroundColor: 'white',
        }}
      >
        Future is in your subscription
      </div>
    </div>
  )
}
