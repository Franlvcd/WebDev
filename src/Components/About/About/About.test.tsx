import ReactDOM from 'react-dom'
import { About } from '../About'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<About />, div)
})

it('renders title correctly', () => {
  const { getByTestId } = render(<About />)
  expect(getByTestId('title-about')).toHaveTextContent('About Us')
})

it('matches snapshot', () => {
  const tree = renderer.create(<About />).toJSON()
  expect(tree).toMatchSnapshot()
})
