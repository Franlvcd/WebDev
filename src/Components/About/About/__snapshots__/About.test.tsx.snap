// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundImage": "url(\\"https://www.britannica.com/explore/savingearth/wp-content/uploads/sites/4/2019/03/environmental-engineering-hero.jpg\\")",
        "backgroundPosition": "center",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "cover",
        "minHeight": 500,
        "padding": 0,
      }
    }
  />
  <div
    data-testid="title-about"
  >
    <p
      className="title-text-about"
    >
      About Us
    </p>
    <p
      className="about-content"
    >
      We are a group of people who are wildly passionate about changing the way engineering solution is built. We believe jumping out of the box and thinking about a bigger picture - the globe and future generations. We exist to embrace the global ecosystem and unlock creativity. Our platform empowers the world's leading firms to build eco-fiendly solutions.
    </p>
  </div>
  <div
    className="background-vision"
  >
    <p
      className="vision-text"
    >
      Our Vision
    </p>
    <p
      className="about-content"
    >
      Bring inspiration and innovation to people with our expertise in envirimental engineering.
    </p>
  </div>
</div>
`;
